### PR TITLE
[REF] mail: `MobileMessagingNavbar`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2837311

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/components/mobile_messaging_navbar/mobile_messaging_navbar.scss [hunk 1]
```diff
@@ -3,41 +3,13 @@
...
-    font-size: 0.8em;
-}
-
-// ------------------------------------------------------------------
-// Style
-// ------------------------------------------------------------------
-
-.o_MobileMessagingNavbar {
-    background-color: $white;
-    box-shadow: 0 0 8px gray('400');
```
Comment (Brieuc-brd) [inline addons/mail/static/src/components/mobile_messaging_navbar/mobile_messaging_navbar.scss:34]: This custom shadow is replaced by `border-top shadow-lg`

File: addons/mail/static/src/components/mobile_messaging_navbar/mobile_messaging_navbar.scss [hunk 2]
```diff
@@ -3,41 +3,13 @@
...
-}
-
-.o_MobileMessagingNavbar_tab {
-    box-shadow: 1px 0 0 gray('400');
```
Comment (Brieuc-brd) [inline addons/mail/static/src/components/mobile_messaging_navbar/mobile_messaging_navbar.scss:38]: This custom shadow, used to simulate a border, is replaced by `border-left`.

**Comment by Brieuc-brd** [comment]
Hello @alexkuhn 👋 
This PR is ready for review.

cc. @stefanorigano

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**